cms/templates/index.html

```diff
@@ -443,10 +443,25 @@ <h3 class="course-title">${library_info['display_name']}</h3>
       <div class="notice notice-incontext notice-instruction notice-instruction-nocourses list-notices libraries-tab">
         <div class="notice-item">
           <div class="msg">
```

Comment:
@marcotuts Does this mean I should be checking for course authoring privileges when displaying the create button?